src/error.rs

```diff
@@ -44,14 +44,14 @@ impl From<ParseError> for RequestError {
 
 impl From<io::Error> for RequestError {
     fn from(e: io::Error) -> Self {
-        RequestError::HyperError(HyperError::from(e))
+        unimplemented!()
```

Comment:
I'll write the error handling code if needed but we cannot just push it into `ReqwestError` as it was done with hyper before, because the relevant API is not public.

I'm wondering if we should just unwrap in the only place where an io error might occur, which is line 53 in request.rs, or add a whole variant just for this which is not supposed to happen anyway?

Reply:
That's a good point - the error can't happen since we're using a `Vec<u8>` as a `Write` implementor. You can remove this impl and `.unwrap()` that call (with a comment indicating why the unwrap never fails).